ivy/utils/_importlib.py

```diff
@@ -89,6 +158,14 @@ def _import_module(name, package=None):
         parent_name, _, child_name = absolute_name.rpartition(".")
         parent_module = _import_module(parent_name)
         path = parent_module.__spec__.submodule_search_locations
+
+    # Return the one from global Ivy if the module is marked to skip
+    if absolute_name in _all_modules_to_skip:
```

Comment:
```python
    for module in MODULES_TO_SKIP:
        # the first condition alone is not enough, the second condition ensures that the module is indeed "module_to_skip.*"
        if absolute_name in module and len(absolute_name) >= len(module):
            if path is not None:
                # Set reference to self in parent, if exist
                setattr(parent_module, child_name, sys.modules[absolute_name])
            return sys.modules[absolute_name]
```
I think something like this could work, also, we are assuming the module is already imported in `sys.modules`, is this safe to assume?